openmetadata-spec/src/main/resources/json/schema/api/services/createDatabaseService.json

```diff
@@ -31,6 +31,6 @@
       "$ref": "../../type/entityReference.json"
     }
   },
-  "required": ["name", "serviceType", "connection"],
+  "required": ["name", "serviceType"],
```

Comment:
now since we are passing the connection anyways let's not remove this from required